001-quickstart-passwordless/Program.cs

```diff
@@ -0,0 +1,78 @@
+﻿// ------------------------------------------------------------
+// Copyright (c) Microsoft Corporation.  All rights reserved.
+// ------------------------------------------------------------
+
+// <using_directives> 
+using Microsoft.Azure.Cosmos;
+using Azure.Identity;
+// </using_directives>
+
+// <client_credentials> 
+// New instance of CosmosClient class
+using CosmosClient client = new(
+    "https://<your-cosmosdb-account-name>.documents.azure.com:443/",
+    new DefaultAzureCredential()
+);
+// </client_credentials>
+
+// <new_database> 
+// Database reference with creation if it does not already exist
+Database database = client.GetDatabase("<your-database-name>");
```

Comment:
```suggestion
Database database = client.GetDatabase(
    id: "cosmicworks"
);
```

In the [current Quickstart](https://learn.microsoft.com/en-us/azure/cosmos-db/nosql/quickstart-dotnet?tabs=azure-portal%2Cwindows#create-a-database), we're prescriptive about the name here.